ui/src/app/views/dataset-page.tsx

```diff
@@ -162,12 +159,14 @@ export const ValueListItem: React.FunctionComponent <
     </div>;
   };
 
-const plusLink = (dataTestId: string, path: string) => {
-  return <a data-test-id={dataTestId} href={path}
+const plusLink = (dataTestId: string, path: string, disable: boolean) => {
+  return <TooltipTrigger data-test-id='plus-icon-tooltip' disabled={!disable}
+                         content='Requires Owner or Writer permission'>
+    <Clickable disabled={disable} data-test-id={dataTestId} href={path}
             onClick={e => {navigateAndPreventDefaultIfNoKeysPressed(e, path); }}>
     <ClrIcon shape='plus-circle' class='is-solid' size={16}
-             style={styles.selectBoxHeaderIconLinks}/>
-  </a>;
```

Comment:
Does moving from `<a>` to `<Clickable>` have any screen reader/href/link implications? If you cmd click the icon, does it open in a new tab still? I think we can keep the Clickable as an `<a>`

Reply:
It opens it in the same tab. I wanted to keep it consistent  with the data tab page.

Reply:
Okay, that seems fine for now. We should audit our components and see what makes sense moving forwards though.